lib/os-cluster-entrypoint.ts

```diff
@@ -151,6 +151,9 @@ export class OsClusterEntrypoint {
       const use50heap = `${scope.node.tryGetContext('use50PercentHeap')}`;
       const use50PercentHeap = use50heap === 'true';
```

Comment:
Thanks @rishabh6788 , where are we using this `use50PercentHeap` value?

Reply:
This is optional parameter, processed [here](https://github.com/opensearch-project/opensearch-cluster-cdk/blob/main/lib/infra/infra-stack.ts#L474-L483). @prudhvigodithi

Reply:
it was included due to formatting fix.